controlplane/api/local/connection/connection_helpers.go

```diff
@@ -137,3 +137,11 @@ func (c *Connection) IsComplete() error {
 
 	return nil
 }
+
+func (c *Connection) GetSignature() string {
+	return c.GetResponseJWT()
```

Comment:
As I know JWT has 3 parts:
Header
Payload
Signature

Is this method should return part of JWT or can we rename it?

Reply:
The name of that method doesn't imply dependency on some specific technology that being used to implement that signature.